bin/build_cmd.ml

```diff
@@ -188,3 +188,27 @@ let build =
     run_build_command ~common ~config ~request
   in
   (term, Term.info "build" ~doc ~man)
+
+let fmt =
```

Comment:
why is this both in build_cmd and in fmt?

Reply:
I didn't mean to push `fmt.ml`, I removed it.